### DISCUSSION TITLE
Configure NVIDIA variant with time-slicing/shared access

### DISCUSSION THREAD

**Discussion by z0rc** [bottlerocket-os/bottlerocket#2341] [Q&A]
NVIDIA k8s-driver-plugin supports sharing single GPU via time-slicing: https://github.com/NVIDIA/k8s-device-plugin#shared-access-to-gpus-with-cuda-time-slicing

This feature can be enabled and configured via ConfigMap associated with plugin: https://github.com/NVIDIA/k8s-device-plugin#passing-configuration-to-the-plugin-via-a-configmap

I would like to enable time-slicing on Bottlerocket NVIDIA variant. I checked available documentation at main README and https://github.com/bottlerocket-os/bottlerocket/blob/develop/QUICKSTART-EKS.md#aws-k8s--nvidia-variants, but it doesn't mention anything about available configuration options for k8s-driver-plugin or how to pass a config file.

**Comment by arnaldo2792** [comment]
Hello @z0rc, thanks for bringing this to our attention! Currently Bottlerocket doesn't provide a mechanism to change the configurations of the NVIDIA device plugin, I'll create an issue to keep track of this request.

**Comment by arnaldo2792** [comment]
I forgot to add the link for the Github Issue I created to add this feature, but here it is:

https://github.com/bottlerocket-os/bottlerocket/issues/2347
